src/components/search/List.jsx

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+import { useState, useEffect } from 'react';
+
+export const List = ({ inputValue }) => {
+  const [userList, setUserList] = useState();
```

Comment:
```suggestion
  const [userList, setUserList] = useState([]);
```

list라면 초기값도 배열로 주셔야할 것 같습니다!